app/views/layouts/_nav_bar.html.erb

```diff
@@ -3,28 +3,25 @@
     <a class="brand" href="/"><%= image_tag "logo5.png" %></a>
   <% end %>
   <%= menu_group do %>
-      <h3 id="store-name">Keevahh</h3>
+    <h3 id="store-name">Keevahh</h3>
   <% end %>
   <%= menu_group do %>
-    <%= menu_item "Home", root_path %>
-    <% if @tenant %>
-      <%= menu_item "Lend", categories_path(tenant: @tenant.organization) %>
-    <% else %>
-      <%= menu_item "Lend", root_path %>
-    <% end %>
-    <%= menu_item "Pending Loans", pending_loan_path %>
+    <%= menu_item "Lend", categories_path %>
```

Comment:
Maybe we want to add an if here to have this disappear if current_user && current_user.admin? (either in this PR or in a new ticket).